mantidproject/mantid
# Add system tests for ISIS Powder scripts

Description of work.
This PR adds several system tests for the `focus` and `create_vanadium` methods of the ISIS Powder scripts. It also fixes one or two minor things picked up by the new tests too.

I also tided out all the old files from the previous system tests which have been removed for a while now.

**To test:**
Code review
Ensure new system tests are run and pass

Fixes #19794 .

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
